Changed
^^^^^^^

- The `funcx` error module has been renamed from `funcx.utils.errors` to
  `funcx.errors`
